scanner/updater/rhel/parser.go

```diff
@@ -140,6 +140,7 @@ func severity(ctx context.Context, def oval.Definition) string {
 
 	// For CVEs, there will only be 1 item in this slice.
 	// For RHSAs, RHBAs, etc., there will typically be 1 or more.
+	var cveId string
```

Comment:
Something to keep in mind: the VEX work https://github.com/stackrox/stackrox/pull/12452 will obsolete this, so don't stress too much about this.